src/components/Header/Header.tsx

```diff
@@ -1,11 +1,36 @@
-import React from 'react';
-import { NavLink } from 'react-router-dom';
-
-export const Header = () => (
-  <nav>
-    <NavLink to="/">Home</NavLink>
-    <NavLink to="/phones">Phones</NavLink>
-    <NavLink to="/tablets">Tablets</NavLink>
-    <NavLink to="/accessories">Accessories</NavLink>
-  </nav>
-);
+import { FC, useState } from 'react';
+import { Link } from 'react-router-dom';
+import './header.scss';
+import { Burger } from '../Burger';
+import { MenuLinks } from '../MenuLinks';
+import { ActionsLink } from '../ActionLinks';
+import BurgerMenu from '../../img/icons/burger.svg';
+import NiceGadgetsLogo from '../../img/icons/nice_gadgets.svg';
+
+export const Header: FC = () => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+  const handleMenuClick = () => {
+    setIsMenuOpen(!isMenuOpen);
+  };
+
+  return (
+    <>
+      <header className="header">
+        <div className="header__nav">
+          <Link className="header__logo" to="/">
+            <img src={NiceGadgetsLogo} alt="logo" />
+          </Link>
+
+          <MenuLinks />
+        </div>
+
+        <ActionsLink />
+        <Link to="#burger" className="burger__menu" onClick={handleMenuClick}>
+          <img src={BurgerMenu} alt="burger_menu" />
+        </Link>
```

Comment:
If we don't need router functionality here, only onClick handler, it's better to use `<button>` tag